modelscope/preprocessors/nlp/word_alignment_preprocessor.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+
+import os
+import os.path as osp
+from typing import Any, Dict, Optional, Union
+import itertools
+import numpy as np
+import torch
+
+from modelscope.metainfo import Preprocessors
+from modelscope.preprocessors.base import Preprocessor
+from modelscope.preprocessors.builder import PREPROCESSORS
+from modelscope.utils.constant import Fields, ModeKeys
+from modelscope.utils.hub import get_model_type
+from modelscope.utils.logger import get_logger
+from .transformers_tokenizer import NLPTokenizer
+from transformers import AutoTokenizer
+    
+@PREPROCESSORS.register_module(
+    Fields.nlp, module_name=Preprocessors.word_alignment)
+class WordAlignmentPreprocessor(Preprocessor):
+    """The tokenizer preprocessor used in sentence embedding.
+    """
+
+    def __init__(self,
+                 model_dir: str,
+                 sequence_pair='sentence_pair',
+                 mode=ModeKeys.INFERENCE,
+                 use_fast: bool = False,
+                 max_length: int = None,
+                 **kwargs):
+        """The preprocessor for sentence embedding task, based on transformers' tokenizer.
+
+        Args:
+            model_dir: The model dir used to initialize the tokenizer.
+            first_sequence: The key of the first sequence.
+            second_sequence: The key of the second sequence.
+            mode: The mode for the preprocessor.
+            use_fast: Use the fast tokenizer or not.
+            max_length: The max sequence length which the model supported,
+                will be passed into tokenizer as the 'max_length' param.
+            **kwargs: Extra args input into the tokenizer's __call__ method.
```

Comment:
add introduction for arg sequence_length

Reply:
I have updated pipline and preprocessor, please try again, thx.

Reply:
Hi, could you do me a favor? In fact, i did fix the above two problems according to your suggestion, but i didn't konw why all checks still failed.

Reply:
the linter test have passed,  the citest failed because some model configuration have been updated, please merge the newest master code which is updated a few minitues ago

Reply:
thanks, i have fixed conflicts and merged the newest master code.